client/src/test/java/se/fortnox/reactivewizard/client/HttpClientConfigTest.java

```diff
@@ -83,6 +86,14 @@ void shouldSetPort() throws URISyntaxException {
         assertThat(config.getHost()).isEqualTo("localhost");
     }
 
+    @Test
+    void shouldSetRoot() throws URISyntaxException {
+		assertThat(new HttpClientConfig("http://localhost:8080").getRoot()).isEqualTo("");
```

Comment:
Missing indentation here.

Reply:
How so, it looks identical to indentations above and below?

Reply:
![image](https://github.com/FortnoxAB/reactive-wizard/assets/66864507/69cd77d4-1f23-48ee-8527-bf60c5cb751d)
compared to ex.
![image](https://github.com/FortnoxAB/reactive-wizard/assets/66864507/e4ecd830-6dcb-46e7-b786-4c65608162d1)

Reply:
Hmm, this is what I see:

![bild](https://github.com/FortnoxAB/reactive-wizard/assets/1902009/6f5e3fc9-4869-418b-b2bd-f32f33f1e7da)